OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -76,6 +76,9 @@ public class MissileInfo : IProjectileInfo
 		[Desc("Vertical rate of turn.")]
 		public readonly int VerticalRateOfTurn = 6;
 
+		[Desc("Gravity applied while in free fall.")]
```

Comment:
I felt that negative values are unintuitive for most 'end-users'/modders (negative gravity means "falling" towards the sky in my book).